<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddSensForm</class>
 <widget class="QDialog" name="AddSensForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Sens</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>121</width>
     <height>99</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="labelPort">
      <property name="text">
       <string>Port: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelVal">
      <property name="text">
       <string>Value: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="valEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="portEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelFlag">
      <property name="text">
       <string>Flag: </string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QCheckBox" name="flagBox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelAct">
      <property name="text">
       <string>Action: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="actTypeBox"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
